staging/src/k8s.io/component-base/metrics/prometheus/controllers/metrics.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"sync"
+
+	k8smetrics "k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+var (
+	once sync.Once
+)
+
+// ControllerMetrics includes all the metrics of the proxy server.
+type ControllerMetrics struct {
+	controllerInstanceCount *k8smetrics.GaugeVec
+}
+
+// NewControllerMetrics create a new ControllerMetrics, configured with default metric names.
+func NewControllerMetrics() *ControllerMetrics {
+	controllerInstanceCount := k8smetrics.NewGaugeVec(
+		&k8smetrics.GaugeOpts{
+			Name:           "managed_controller_started",
```

Comment:
I prefer `running_managed_controllers`.

`_started` is technically inaccurate since this is a gauge and therefore can presumably go down...